virt/virt-2-6-release-notes.adoc

```diff
@@ -59,8 +59,9 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //CNV-9455: Improvements to KubeVirt and CDI ensure that storage is allocated on the correct node
 
-//CNV-9456: When importing VM disk images, you can now control which multus network is used
+* While a disk image is prepared for consumption by a virtual machine, it is possible that the virtual machine cannot be scheduled to the node where the local storage PV was previously pinned. In this release, `StorageClass` with `volumeBindingMode` is set to `WaitForFirstConsumer`, and the binding and provisioning of the PV is delayed until a `Pod` is created using the PVC.
```

Comment:
For this one I'd focus more on the apparent impact to the user rather than the technical details.  For example:

When preparing local storage for a virtual machine disk image a volume may be allocated from a different node than is required by the VM.  To prevent scheduling problems, this release integrates with the kubernetes WaitForFirstConsumer binding mode to ensure that volumes are allocated from the correct node.

Reply:
Thanks Adam, that's much better. Appreciate the big picture view, I wasn't getting it. Moving to QE Review.